docker/transformers-pytorch-deepspeed-latest-gpu/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel-23-11.html#rel-23-11
-FROM nvcr.io/nvidia/pytorch:23.11-py3
+FROM nvcr.io/nvidia/pytorch:23.04-py3
```

Comment:
any > 23.05 with torch 2.2 below will fail the build. The error message has no useful information but 

> The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.

cc @muellerzr